docs/products/access/topics/look-and-feel/translations/translations.md

```diff
@@ -32,27 +32,27 @@ property files to the template set in your git repository to make use of the cus
 further customize messages. These messages are used by the [default templates](../templates/templates.md).
 
 ```
- - messages.properties
- - messages_locale.properties
+ - messages-engine.properties
+ - messages-engine_locale.properties
```

Comment:
I checked in access, and it shouldn't matter how files are named as long as they end with `_locale`.
But It's good to have the examples aligned with the zip file.